.github/workflows/pr.yaml

```diff
@@ -146,7 +146,7 @@ jobs:
       build_type: pull-request
       script: "ci/test_thirdparty.sh"
       matrix: ${{ needs.compute-matrix.outputs.TEST_MATRIX }}
-      matrix_filter: map(select(.ARCH == "amd64" and (.CUDA_VER | split(".") | .[0] | tonumber >= 12))) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
+      matrix_filter: map(select(.ARCH == "amd64" and (.CUDA_VER | split(".") | .[0] | tonumber == 12))) | group_by(.CUDA_VER|split(".")|map(tonumber)|.[0]) | map(max_by([(.PY_VER|split(".")|map(tonumber)), (.CUDA_VER|split(".")|map(tonumber))]))
```

Comment:
Do we need a TODO to re-enable tests with CUDA 13? What requirements are missing? (e.g. cupy, …?)

Reply:
The thirdparty test is presently cuDF, so having a version of cuDF that officially supports CUDA 13 is probably a good criteria for this - I think there's not a CUDA 13-supporting version yet?

Reply:
Correct. We are waiting on numba-cuda. 🙃 We'll get this sorted out soon enough.